pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -388,6 +397,75 @@ class OSRow extends React.Component {
     }
 }
 
+const UnattendedRow = ({ validationFailed, unattendedDisabled, unattendedInstallation, os, onValueChanged }) => {
+    const validationStateRootPassword = validationFailed.rootPassword ? 'error' : undefined;
+    const validationStateUserPassword = validationFailed.userPassword ? 'error' : undefined;
+    let unattendedInstallationCheckbox = (
+        <label className="checkbox-inline">
+            <input id="unattended-installation" type="checkbox"
+                checked={unattendedInstallation}
+                disabled={unattendedDisabled}
+                onChange={e => onValueChanged('unattendedInstallation', e.target.checked)} />
+            {_("Run unattended installation")}
+            <FieldLevelHelp content={_("Perform an unattended install using libosinfo's install script support")} close />
+        </label>
+    );
+    if (unattendedDisabled) {
+        unattendedInstallationCheckbox = (
+            <OverlayTrigger overlay={ <Tooltip id='os-unattended-installation-tooltip'>{ _("The selected Operating System does not support unattended installations") }</Tooltip> } placement='left'>
+                {unattendedInstallationCheckbox}
+            </OverlayTrigger>
+        );
+    }
+
+    return (
+        <>
+            {unattendedInstallationCheckbox}
+            {!unattendedDisabled && unattendedInstallation ? <>
+                {os.profiles.length > 0 ? <>
+                    <label className="control-label" htmlFor="profile-select">
+                        {_("Profile")}
+                    </label>
+                    <Select.Select id="profile-select"
+                        initial={os.profiles && os.profiles[0]}
+                        onChange={e => onValueChanged('profile', e)}>
+                        { (os.profiles || []).map(profile => {
+                            let profileName;
+                            if (profile == 'jeos')
+                                profileName = 'Server';
+                            else if (profile == 'desktop')
+                                profileName = 'Workstation';
+                            else
+                                profileName = profile;
+                            return <Select.SelectEntry data={profile} key={profile}>{profileName}</Select.SelectEntry>;
+                        }) }
+                    </Select.Select>
+                </> : null }
```

Comment:
is there any advantage of doing `{condition ? < ... > : null}` rathern than `{condition && < ... >}` . No need to change it, just asking.

Reply:
There is a reason - yes. 
It will show in the UI the value of the condition if you write it like that.